www/artists/afyfe/gallery.html

```diff
@@ -24,24 +24,27 @@
             </div>
             <div id="gallery">
                 <p>
-                    <a href="mossyrocks.html"><img src="images/mossyrocks.jpg" alt="Mossy Rocks" class='b1'></a> <a href="natureresting.html"><img src="images/natureresting.jpg" alt="Nature Resting" class='b1'></a>
+                    <a href="mossyrocks.html"><img src="images/mossyrocks.jpg" alt="Mossy Rocks" class="b1"></a> <a href="natureresting.html"><img src="images/natureresting.jpg" alt="Nature Resting" class="b1"></a>
                     <br>
-                    Mossy Rocks | Nature Resting
+                    Mossy Rocks | Nature Resting (psychedelic ambiguity of nature and self.)
                 </p>
                 <p>
-                    <a href="atthebeach.html"><img src="images/atthebeach.jpg" alt="At the Beach" class='b1'></a> <a href="morning.html"><img src="images/morning.jpg" alt="Morning" class='b1'></a>
+                    <a href="atthebeach.html"><img src="images/atthebeach.jpg" alt="At the Beach" class="b1"></a> <a href="morning.html"><img src="images/morning.jpg" alt="Morning" class="b1"></a>
                     <br>
                     On the Beach | Morning
                 </p>
                 <p>
-                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class='b1'></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class='b1'></a>
+                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class="b1"></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class="b1"></a>
                     <br>
                     Prarie Lily | The Reflection Nebula
                 </p>
                 <p>
-                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.jpg" width="500" height="375" alt="no alt text"></a>
+                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.JPG" width="500" height="375" alt="no alt text"></a>
                     <br>
-                    Tribute to the Regina 5
+                    (Summarizes the tribute to fyfe's grandfathers generation of artists. Is she cleaning them up? Or wiping them down the drain?) Tribute to the Regina 5
```

Comment:
...You just added what Wayne told you to in the brackets? I'm pretty sure he was asking you to summarize the tribute, not just say "summarizes the tribute".

Reply:
Im not sure but i thought he was meaning that the image summarizes the tribute.

Reply:
You can check with him, but I find it unlikely. The image in question is a link to the gallery with all of the other sinks. Also, why else would he have questions in there at the end?